### PR TITLE
task1: store input

### PR DESCRIPTION
**Pull request by shresoshan**
Added store.py task

### CODE REVIEW

**Comment by sanjaymhj** [comment]
Please add test cases

File: Python/tasks/task1/show.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        results = ''
+        for row in csv_reader:
+            results += "Name: {}".format(row["name"]) + "\n"
+            results += "Date of Birth: {}".format(row["dob"].split(' ')[0]) + "\n"
+            results += row["subject"] + ":\n"
+            results += "\t"+"Score: {}".format(row["score"]) + "\n"
+            results += "\t"+"Total: {}".format(row["total"]) + "\n"
+            results += "Percentage: {}".format(row["percentage"]) + "\n"
+            results += "-----------------"+ "\n\n"
+        csvfile.close()
```
Comment (sagarlf) [inline Python/tasks/task1/show.py:17]: you wouldn't require close, as you are using `with`

**Review by sagarlf — COMMENTED** [review]
Looks good @shresoshan. Only comment is tests and some coding techniques which you can improve as you practice. Will merge it within a day or two.

File: Python/tasks/task1/show.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import os
+import csv
+import argparse
+
+def showresults(fileName):
```
Comment (Basanta78) [inline Python/tasks/task1/show.py:5]:
```suggestion
def show_results(filename):
```

File: Python/tasks/task1/store.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+#Convert to datetime type
+def dob(value):
+    return datetime.strptime(value,'%d/%m/%Y')
+
+#Calculate percentage
+def calcpercentage(total, score):
+    return ((score/total)*100)
+
+#Duplicate Check
+def isDuplicate(args):
```
Comment (Basanta78) [inline Python/tasks/task1/store.py:17]:
```suggestion
def is_duplicate(args):
```

Comment (Basanta78) [reply Python/tasks/task1/store.py:17]: Follow snake case for all the functions and variables

**Pull request merged**